server/templates/pages/src/400.html

```diff
@@ -1,7 +1,5 @@
 <!doctype html>
-<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html class="lt-ie9"> <![endif]-->
+<!--[if lt IE 9]>      <html class="no-js"> <![endif]-->
```

Comment:
The overlap in versions in the conditional comments is more difficult than need be to reason about and could be made more clear. I would change this and the following line to:

```html
<!--[if lte IE 8]>      <html class="no-js"> <![endif]-->
<!--[if gte IE 9]><!--> <html class="no-js" dir="{{ lang_dir }}" lang="{{ lang }}"> <!--<![endif]-->
```

We should also ensure these pages are at least readable in IE8/IE9.